assets/scss/optimized-checkout.scss

```diff
@@ -1 +1,111 @@
 // checkout scss file
+@import 'settings/settings';
+@import 'tools/tools';
+@import 'generic/generic';
+@import 'elements/elements';
+@import 'vendor/vendor';
+@import 'objects/objects';
+@import 'components/components';
+@import 'utilities/utilities';
+
+@mixin form-input() {
+    box-shadow: none;
+}
+
+.layout,
```

Comment:
Please add classname prefix if possible e.g. `.c-...`

Reply:
@aleksandraworhacz The checkout HTML content is rendered by BigCommerce and we have no control over it. We can't change the classnames or HTML.

(unless we use the BigCommerce Checkout SDK, but thats another story 😄  )

Reply:
I thought so, but it was worth to ask :)